src/shared_utilities/DataHandling.jl

```diff
@@ -0,0 +1,246 @@
+module DataHandling
+
+using Dates
+
+import ClimaCore
+
+import ..FileReader: NCFileReader
+import ..Regridder
+import ..Regridder:
+    AbstractRegridder, TempestRegridder, regrid, AVAILABLE_REGRIDDERS
+
+"""
+
+Note: It is best to have one file per variable with all the temporal data.
+
+Assumptions:
+- There is only one file with all the data in time
```

Comment:
this seems to contradict the above Note?

Reply:
It's probably poorly worded: instead of having multiple files with ranges of dates, it is best to have one single file with alle the dates ("all the data (for a given variable) in time").